flink-table/flink-table-planner/src/test/resources/org/apache/flink/table/planner/plan/stream/sql/SetOperatorsTest.xml

```diff
@@ -231,10 +231,8 @@ LogicalMinus(all=[false])
     </Resource>
     <Resource name="optimized exec plan">
       <![CDATA[
-GroupAggregate(groupBy=[c], select=[c])
-+- Exchange(distribution=[hash[c]])
-   +- Calc(select=[c])
-      +- TableSourceScan(table=[[default_catalog, default_database, T1]], fields=[a, b, c])
+Calc(select=[c])
```

Comment:
Is this change due to the introduction of the PruneEmptyRule about GroupAgg? It seems different from before. If I understand correctly, the previous GroupAggregate also implied deduplication, right?

Reply:
@xuyangzhong yes